build.gradle

```diff
@@ -114,20 +117,13 @@ task copyMasterSheets(type: Copy) {
     include 'psheet_fantasy_std.htm'
     from 'outputsheets/d20/fantasy/htmlxml'
     into 'outputsheets/d20/historical/htmlxml'
-
-    into ('../../horror/htmlxml') {
-	    from 'outputsheets/d20/fantasy/htmlxml'
-    }
-    into ('../../sciencefiction/htmlxml') {
-	    from 'outputsheets/d20/fantasy/htmlxml'
-    }
-    into ('../../western/htmlxml') {
-	    from 'outputsheets/d20/fantasy/htmlxml'
-    }
+    into 'outputsheets/d20/horror/htmlxml'
```

Comment:
This was one of those areas I had trouble with when initially implementing the build and had to use some unusual workarounds. 

I've just tested these changes out using the gradle wrapper (v2.2) and copymastersheets doesn't copy the files across. It may well work in later versions of gradle, but I get a failure in distTar when I try the build in gradle 2.6 or 2.10.